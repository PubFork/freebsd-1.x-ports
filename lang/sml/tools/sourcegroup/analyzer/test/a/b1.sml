functor F (A:A) = struct
  val c = C.z
  open A
  val b = B.x
  val d = D.y
end

structure F = F(A)
