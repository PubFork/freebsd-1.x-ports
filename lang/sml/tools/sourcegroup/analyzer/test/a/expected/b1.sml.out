source sml "a/b1.sml"
  import structure A B C D 
  import signature A 
  export structure F 
  export functor F ;

